app.js

```diff
@@ -80,7 +89,29 @@ angular.module('PathOfDamage', [])
       hit *= (1 - lessTaken.value / 100)
     });
 
-    return Math.round(hit || 0) + ' (' + Math.round(shifted) + ')';
+    return {
+      hit: initialHit,
+      taken: Math.round(hit || 0),
+      shifted: Math.round(shifted),
+      remaining: Math.round($scope.health - hit || 0)
+    }
+  }
+
+  $scope.updateHits = function(skipStringify) {
```

Comment:
I think we need to split the updateHit method up at this point. It was originally designed to be called when a hit entry was edited, which is why it checks if it needs to delete or add rows to the table. There's no real use case for looping through each hit and checking that condition for every entry. Maybe we need updateHitTable and updateHitCalc or something?